sdk/core/core-tracing/lib/wrappers/opencensus/openCensusSpanWrapper.ts

```diff
@@ -101,7 +101,7 @@ export class OpenCensusSpanWrapper implements Span {
   addLink(spanContext: SpanContext, attributes?: Attributes): this {
     // Since there is no way to specify the link relationship
     // It is set as Unspecified = 0
-    this._span.addLink(spanContext.traceId, spanContext.spanId, LinkType.UNSPECIFIED, attributes as OpenCensusAttributes);
+    this._span.addLink(spanContext.traceId, spanContext.spanId, 0, attributes as OpenCensusAttributes);
```

Comment:
please put a comment like `/* LinkType.UNSPECIFIED */` after the 0 so folks know what it means -- actually I think I do this one for you after you rebase. :)

Reply:
Yeah, since I need to rebase on your pending PR anyway, I'll accept your changes over mine.